media-video/qmplay2/qmplay2-18.04.01.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="de es fr hu pl ru uk zh"
+
+inherit cmake-utils l10n xdg
+
+MY_PN="QMPlay2"
```

Comment:
Make it `MY_P`, and include the version and `-src-`. Then you would reduce repetition in `SRC_URI` and `S`. Also preferably don't use `MY_PN` inside URL prefix, so that people can copy-paste it up to `download/`.